scripts/apply-cleaned-db-to-target.sh

```diff
@@ -5,12 +5,12 @@ echo "Target is: ${TARGET}"
 
 mkdir -p ./dumps
 LATEST_PROD_DUMP=$(aws s3 ls digitalmarketplace-database-backups | grep production | sort -r | head -1 | awk '{print $4}')
-pg_dump --no-acl --no-owner --clean postgres://postgres:@localhost:63306/postgres | gzip > ./dumps/cleaned-"${LATEST_PROD_DUMP%.*}"
+pg_dump --no-acl --no-owner --clean postgres://postgres:@localhost:63306/postgres | gzip > ./dumps/cleaned-"${LATEST_PROD_DUMP}"
 
```

Comment:
On jenkins:
```
> aws s3 ls digitalmarketplace-database-backups | grep production | sort -r | head -1 | awk '{print $4}'
production-201903190300.sql.gz.gpg
```

The file is brought down from S3 with the timestamp attached. 
The `%.*` strips the `.gpg` suffix so it becomes:
```
production-201903190300.sql.gz.gpg
```

This means we'll be uploading the file as `production-201903190300.sql.gz` which won't action the versioning as it'll have a different name in the bucket.